lib/tools/system-calls.js

```diff
@@ -398,6 +401,7 @@ systemCallMethods.adbExec = async function adbExec (cmd, opts = {}) {
   // setting default timeout for each command to prevent infinite wait.
   opts.timeout = opts.timeout || this.adbExecTimeout || DEFAULT_ADB_EXEC_TIMEOUT;
   opts.timeoutCapName = opts.timeoutCapName || 'adbExecTimeout'; // For error message
+  opts.outputFormat = opts.outputFormat || 'stdout';
```

Comment:
can we do this without mutating `opts`?

Reply:
Sure, now setting new `outputFormat` value here instead of mutating `opts`